ocfweb/atool/chpass/forms.py

```diff
@@ -0,0 +1,93 @@
+import ocflib.account.search as search
+import ocflib.account.validators as validators
+import ocflib.ucb.groups as groups
+from django import forms
+
+from ocfweb.atool.constants import TEST_OCF_ACCOUNTS
+from ocfweb.atool.constants import TESTER_CALNET_UIDS
+from ocfweb.atool.utils import wrap_validator
+
+
+def _get_accounts_signatory_for(calnet_uid):
+    def flatten(lst):
+        return [item for sublist in lst for item in sublist]
+
+    group_accounts = flatten(map(
+        lambda group: group['accounts'],
+        groups.groups_by_student_signat(calnet_uid).values()))
+
+    # sanity check since we don't trust CalLink API that much:
+    # if >= 10 groups, can't change online, sorry
+    if len(group_accounts) < 10:
+        return group_accounts
+    return []
```

Comment:
nit: this should raise an error rather than silently not show accounts (maybe just `assert len(group_accounts) < 10`).